include/pistache/http.h

```diff
@@ -613,31 +611,35 @@ namespace Private {
         Message *message;
     };
 
-    struct RequestLineStep : public Step {
```

Comment:
Why are you changing the struct to a class?  What warning (and in what compiler version) is this change addressing?

Reply:
This is addressing the same issue as the `ConnectionPool`.